docs/core/resilience/snippets/http-resilience/Program.RemoveHandlers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Http.Resilience;
+using Polly;
+
+internal partial class Program
+{
+    private static void RemoveHandlers(IHttpClientBuilder httpClientBuilder)
```

Comment:
```suggestion
    private static void RemoveHandlers()
```

This argument can be removed because it is not used.